irohad/ametsuchi/impl/postgres_query_executor.cpp

```diff
@@ -241,21 +189,14 @@ namespace iroha {
 
         return apply(
             viewPermissions<PermissionTuple>(range.front()),
-            [this,
-             range,
-             &response_creator,
-             err_response =
-                 std::forward<ErrResponse>(err_response)](auto... perms) {
+            [this, range, &response_creator, &err_response](auto... perms) {
               bool temp[] = {not perms...};
               if (std::all_of(std::begin(temp), std::end(temp), [](auto b) {
                     return b;
                   })) {
                 return logAndReturnErrorResponse(
                     QueryErrorType::kStatefulFailed,
-                    err_response(),
-                    this->query_hash_,
-                    query_response_factory_,
-                    log_);
+                    std::forward<std::string>(err_response()));
```

Comment:
It should be `std::forward<ErrResponse>(err_response)()`, since return type is `std::string` value anyway, but `err_response` can be reference or value.